Client/Frontend/Strings.swift

```diff
@@ -355,6 +364,7 @@ extension String {
         public struct Tabs {
             public static let TabsSectionTitle = MZLocalizedString("Settings.Tabs.CustomizeTabsSection.Title", value: "Customize Tab Tray", comment: "In the settings menu, in the Tabs customization section, this is the title for the Tabs Tray customization section. The tabs tray is accessed from firefox hompage", lastUpdated: .v39)
             public static let InactiveTabs = MZLocalizedString("Settings.Tabs.CustomizeTabsSection.InactiveTabs", value: "Inactive Tabs", comment: "In the settings menu, in the Tabs customization section, this is the title for the setting that toggles the Inactive Tabs feature, a separate section of inactive tabs that appears in the Tab Tray, on or off", lastUpdated: .v39)
+            public static let InactiveTabsDescription = MZLocalizedString("Settings.Tabs.CustomizeTabsSection.InactiveTabsDescription.v101", value: "Tabs you haven’t viewed for two weeks get moved to the inactive section.", comment: "In the settings menu, in the Tabs customization section, this is the description for the setting that toggles the Inactive Tabs feature, a separate section of inactive tabs that appears in the Tab Tray, on or off", lastUpdated: .v101)
```

Comment:
I think we should break up this part of the comment: `, a separate section of inactive tabs that appears in the Tab Tray, on or off", ` Can we put a hard stop before, and then have a new sentence describing what you're saying in this second bit?

Reply:
Sure, I copy pasted the InactiveTabs string description since it had a very similar purpose. I can change that one.